application/dash_application/views/SARS-CoV-2.py

```diff
@@ -123,9 +124,13 @@
 # Build lists of attributes for sorting, shaping, and filtering on
 ALL_PROJECTS = util.unique_set(BEDTOOLS_DF, PINERY_COL.StudyTitle)
 ALL_KITS = util.unique_set(BEDTOOLS_DF, PINERY_COL.PrepKit)
-# TODO: Remove kludge and uncomment real line after Pinery is merged
-ILLUMINA_INSTRUMENT_MODELS = util.unique_set(BEDTOOLS_DF, PINERY_COL.InstrumentName)
-#ILLUMINA_INSTRUMENT_MODELS = list(util.get_illumina_instruments(RNA_DF))
+
+BEDTOOLS_CALC_DF = util.df_with_instrument_model(BEDTOOLS_CALC_DF, PINERY_COL.SequencerRunName)
+ILLUMINA_INSTRUMENT_MODELS = util.get_illumina_instruments(BEDTOOLS_CALC_DF)
+BEDTOOLS_CALC_DF = BEDTOOLS_CALC_DF[BEDTOOLS_CALC_DF[INSTRUMENT_COLS.ModelName].isin(ILLUMINA_INSTRUMENT_MODELS)]
+# TODO: Do the other dataframes need to be similarly filtered?
```

Comment:
The `BEDTOOLS_PERCENTILE_DF` needs to be merged with the instrument data (has its own `reshape_percentile_df` call as it cannot be cleanly merged with `BEDTOOLS_CALC_DF`.